assessment/collectors/project_analyzer/file_dependency_analyzer.go

```diff
@@ -51,6 +51,10 @@ func validateGoroot() error {
 	return nil
 }
 
+func packagesLoadLogger(format string, args ...interface{}) {
```

Comment:
it is not clear what this method is supposed to do. Can you add docs around it ?

Reply:
It's used to as a logger function by packages.Load library.